manifest.json

```diff
@@ -41,5 +41,13 @@
     "projectOptions": ["app", "addon"],
     "nodeVersion": "6.0.0",
     "commands": ["jscodeshift -t https://raw.githubusercontent.com/simplabs/qunit-dom-codemod/master/qunit-dom-codemod.js ./tests"]
+  },
+  "ember-angle-brackets-codemod": {
+    "versions": {
+      "ember-cli": "3.10.0-beta.1"
+    },
+    "projectOptions": ["app", "addon"],
+    "nodeVersion": "6.0.0",
+    "commands" : ["npx ember-angle-brackets-codemod angle-brackets ./app/templates"]
```

Comment:
npx is implied, so you can remove that part.

Reply:
Done